disperser/cmd/apiserver/main.go

```diff
@@ -114,6 +115,14 @@ func RunDisperserServer(ctx *cli.Context) error {
 		ratelimiter = ratelimit.NewRateLimiter(reg, globalParams, bucketStore, logger)
 	}
 
+	if config.MaxBlobSize < 0 || config.MaxBlobSize > 64*1024*1024 {
+		return fmt.Errorf("configured max blob size is invalid %v", config.MaxBlobSize)
+	}
+
+	if int64(math.Log2(float64(config.MaxBlobSize))) == int64(math.Log2(float64(config.MaxBlobSize-1))) {
```

Comment:
It's faster to use bit operation, `config.MaxBlobSize > 0 && (config.MaxBlobSize&(config.MaxBlobSize-1)) == 0` should work

Reply:
this only runs once. I think it is more readable this way

Reply:
but your solution works

Reply:
I think there are utility functions to check if a number is power of 2 (in encoding or common package). Reusing that has no impact on readabilty.

Reply:
that function is in "github.com/Layr-Labs/eigenda/encoding/fft". I don't want to add a new strange dependancy, what do you think

Reply:
ideally we can pull it out to `common/` and share across the codebase